### ISSUE TITLE
Set up security policy

### ISSUE THREAD

**Issue by l0b0** [linz/geospatial-data-lake#131]
See [the policy page](https://github.com/linz/geospatial-data-lake/security/policy); we should at least have a documented way to report vulnerabilities.

**Comment by billgeo** [comment]
Hey. Can you explain why we would want to do this? Doesn't seem to be a pattern for it at LINZ? None of the open repos or Landonline search (a closed one by very high profile security for LINZ), appear to do this?

**Comment by l0b0** [comment]
From the linked page:

> Help your community understand how to securely report security vulnerabilities for your project.

For example, reporting vulnerabilities via plaintext email would not be very secure.

**Comment by billgeo** [comment]
So this is for people outside of LINZ to report vulnerabilities to us?

**Comment by l0b0** [comment]
Yep, that's the idea. Someone might notice that something is public which shouldn't be (for example), and may want to notify us without anyone else finding out.
